src/modules/0.0.5/aws_ecs_simplified/cloud_create_fns.ts

```diff
@@ -0,0 +1,211 @@
+import { Subnet as AwsSubnet, Vpc as AwsVpc } from '@aws-sdk/client-ec2'
+import { CreateLoadBalancerCommandInput } from '@aws-sdk/client-elastic-load-balancing-v2';
+
+import { AWS } from '../../../services/gateways/aws';
```

Comment:
I wanted to do a specific PR for that and told Alejandro to continue with the current pattern.

To break it out and do that right will take some thought to prevent massive duplication of effort.